shared/featureconfig/flags.go

```diff
@@ -36,6 +36,21 @@ var (
 		Name:  "experimental-p2p",
 		Usage: "Use the new experimental p2p library. See issue #3147.",
 	}
+	// UseNewSyncFlag to start the beacon chain using the new sync library.
+	UseNewSyncFlag = cli.BoolFlag{
```

Comment:
@prestonvanloon need to register these flags in the respective main.go

Reply:
No, they are automatically registered https://github.com/prysmaticlabs/prysm/blob/0f123ae562912ba0454f59ed02bdc74df4e909a6/beacon-chain/main.go#L64